### PR TITLE
{vis}[intel/2016a] ImageMagick 7.0.2-9

### PR DESCRIPTION
**Pull request by rtripath89**

### CODE REVIEW

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b0095c06441e440816d264ab6d78a4be for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/14ab2bd13c2b38c3245ab18c3714ab3c for a full test report.

File: easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.0.2-5-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '7.0.2-5'
```
Comment (boegel) [inline easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.0.2-5-intel-2016a.eb]: @rtripath89 tarball for this version as disappeared, please bump to `7.0.2-9`?

Comment (rtripath89) [inline easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.0.2-5-intel-2016a.eb]: @boegel : I have updated the version number, but it looks like they keep replacing versions rather then adding new versions, so this new one would also stop working after some time. It looks like the best way is to use the flag "--try-software-version"

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d740acb76577d0ec49e6726b3683e7d7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/be83135f7f72a58ef8b44cac9f583bc8 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @rtripath89!

**Pull request merged**